src/main/java/de/teamlapen/vampirism/proxy/ClientProxy.java

```diff
@@ -75,6 +77,12 @@ public void renderScreenFullColor(int ticksOn, int ticksOff, int color) {
         if (overlay != null) overlay.makeRenderFullColor(ticksOn, ticksOff, color);
     }
 
+    @Nullable
+    @Override
+    public Entity getMouseOverEntity() {
+        return Minecraft.getMinecraft().objectMouseOver.entityHit;
```

Comment:
Same here, in theory `objectMouseOver` should never be null, but even Vanilla does null check it every time, so we probably should do that too